### DISCUSSION TITLE
Node Flags

### DISCUSSION THREAD

**Discussion by m-alhassan-fadel** [jaredpalmer/razzle#1498] [General] [answered]
How can we pass node js flags (e.g. --inspect) for the server-side in the development?
I tried adding it to the start script like this:
"start": "razzle start --inspect"
but it had no effect!

**Comment by hyeomans** [comment, marked as answer]
https://razzlejs.org/getting-started#debugging-with-inspector

**Reply by m-alhassan-fadel** [reply]
unfortunately, none of the commands there had worked

**Reply by hyeomans** [reply]
did you try:
`"start": "razzle start -- --inspect"`
